projects/laji/src/app/shared-modules/own-submissions/own-submissions.component.ts

```diff
@@ -400,6 +401,12 @@ export class OwnSubmissionsComponent implements OnChanges, OnInit, OnDestroy {
               formID: document.formID,
               form: form.title || document.formID,
               id: document.id,
+              publicityRestrictions:
+                document.publicityRestrictions === Document.PublicityRestrictionsEnum.publicityRestrictionsPublic ?
+                  this.translate.instant('haseka.submissions.publicityRestrictions.public') :
+                    document.publicityRestrictions === Document.PublicityRestrictionsEnum.publicityRestrictionsPrivate ?
+                      this.translate.instant('haseka.submissions.publicityRestrictions.private') :
+                        this.translate.instant('haseka.submissions.publicityRestrictions.protected'),
```

Comment:
I find this a bit unreadable. I'd prefer:

```
publicityRestrictions:
	document.publicityRestrictions === Document.PublicityRestrictionsEnum.publicityRestrictionsPublic
		? this.translate.instant('haseka.submissions.publicityRestrictions.public')
		: document.publicityRestrictions === Document.PublicityRestrictionsEnum.publicityRestrictionsPrivate
			? this.translate.instant('haseka.submissions.publicityRestrictions.private')
			: this.translate.instant('haseka.submissions.publicityRestrictions.protected'),
```

Formatting it this way I find it more clear what is the "question" and what are the outcomes of that question. You can understand nested ternary operations by following the indentation.

(Maybe we should enable a linting rule for this - the way I prefer seems to be called "offset ternary expression" in https://eslint.org/docs/latest/rules/indent)

Reply:
Seems reasonable. One question: Is two spaces indentation enough, or did you include the extra spaces on purpose to the example?

Reply:
In the example I used tabs, which Github apparently display with 8char width. Two spaces is enough in laji-repo :)

Reply:
Alright, fixed now :)